src/main/java/command/DropItem.java

```diff
@@ -0,0 +1,52 @@
+package command;
+
+import dictionary.GameEntities;
+import entity.*;
+
+import java.util.Set;
+
+public class DropItem extends Command {

```

Comment:
Dropping an item could have nice implications on the gameplay. Like dropping honey in the forest to lure out a bear. But if dropping an item is simply to get rid of an item from your inventory(which is infinitely big) then it's a useless feature.